Frameworks/QuartzCore/CALayer.mm

```diff
@@ -569,6 +569,7 @@ - (void)display {
         }
 
         if (!target) {
+#if 0 // TODO(DH): GH#1125 evaluate the interplay between context and displaytexture.
             if ((priv->isOpaque && priv->_backgroundColor == nil) || (priv->backgroundColor.a == 1.0 && 0)) {
```

Comment:
What is going on here? Shouldn't we always be hitting the CreateLayerContentsBitmapContext32() codepath always?